src/axes/axis.ts

```diff
@@ -277,6 +277,8 @@ export class Axis<D> extends Component {
   }
 
   /**
+   * This method is deprecated. Use innerTickLength() instead.
```

Comment:
We are still considering what is the best way to comment on a deprecated method, but I think `@deprecated  As of release 1.3, replaced by innerTickLength()` is the way to go now.